package.json

```diff
@@ -23,7 +23,7 @@
     "@mate-academy/bemlint": "^0.1.1",
     "@mate-academy/eslint-config": "*",
```

Comment:
consider adding files only in src folder by 
'git add src' , do that you don't add unnecessary file like package.json and package-lock.json